src/main/conversions/c2_17_0_2017102601.clj

```diff
@@ -0,0 +1,27 @@
+(ns facepalm.c2-17-0-2017102601
+  (:use [korma.core :only [exec-raw]]))
+
+(def ^:private version
+  "The destination database version"
+  "2.17.0:20171026.01")
+
+(defn- add-cascading-deletions
+  "Adds cascading deletions to the template_attrs table."
+  []
+  (println "\t* adding cascading deletions to the template_attrs table.")
+  (exec-raw "ALTER TABLE ONLY template_attrs DROP CONSTRAINT template_attrs_template_id_fkey")
+  (exec-raw "ALTER TABLE ONLY template_attrs
+             ADD CONSTRAINT template_attrs_template_id_fkey
+             FOREIGN KEY (template_id)
+             REFERENCES templates(id) ON DELETE CASCADE")
+  (exec-raw "ALTER TABLE ONLY template_attrs DROP CONSTRAINT template_attrs_attribute_id_fkey")
+  (exec-raw "ALTER TABLE ONLY template_attrs
+             ADD CONSTRAINT template_attrs_attribute_id_fkey
```

Comment:
Adding the constraints with the raw SQL will work just fine, but you could also re-add them (after dropping both) by using `kameleon.sql-reader/load-sql-file`:
```clojure
(kameleon.sql-reader/load-sql-file "constraints/06_template_attrs.sql")
```
This also tests the changes added to that file in this PR.

Reply:
Sounds reasonable. I'll add that.